language/move-lang/src/expansion/dependency_ordering.rs

```diff
@@ -191,12 +242,28 @@ fn module(context: &mut Context, mident: ModuleIdent, mdef: &E::ModuleDefinition
         .for_each(|sblock| spec_block(context, sblock));
 }
 
-fn struct_def(context: &mut Context, sdef: &E::StructDefinition) {
-    if let E::StructFields::Defined(fields) = &sdef.fields {
-        fields.iter().for_each(|(_, _, (_, bt))| type_(context, bt));
-    }
+//**************************************************************************************************
+// Scripts
+//**************************************************************************************************
+
+fn script_defs(context: &mut Context, scripts: &BTreeMap<String, E::Script>) {
+    scripts
+        .iter()
+        .for_each(|(sname, sdef)| script(context, sname.clone(), sdef))
+}
+
+fn script(context: &mut Context, sname: String, sdef: &E::Script) {
```

Comment:
maybe a comment that scripts cannot affect the dependency graph but the immediate dependencies are collected for the script

Reply:
Comment added